serverjs/cards.js

```diff
@@ -127,8 +127,11 @@ function initializeCardDb(dataRoot, skipWatchers) {
 }
 
 function unloadCardDb() {
-  for (const attribute of Object.values(fileToAttribute)) {
+  for (const [filename, attribute] of Object.entries(fileToAttribute)) {
     delete data[attribute];
+    if (skipWatchers !== true) {
```

Comment:
skip watchers shouldn't permanently unwatch the file?

Reply:
When unloading we need to stop watching because otherwise they sawn reload and the threads watching will never stop. We don't actually have access to skipWatchers here so in #1798 I just wrapped it in a try catch.